container/root/etc/nginx/sites-available/default

```diff
@@ -44,6 +44,15 @@ server {
     fastcgi_pass 127.0.0.1:9000;
   }
 
+  # Allow locally-accessible (only) FPM status page
+  location ~ ^/__status$ {
+    access_log off;
```

Comment:
@ko-be one better. nginx still DOES log on failure: 
![screen shot 2016-10-07 at 11 20 19 am](https://cloud.githubusercontent.com/assets/1202354/19195830/2ac39112-8c81-11e6-8c78-91a07a1545c3.png)